airflow/contrib/example_dags/example_gcp_sql.py

```diff
@@ -29,22 +29,32 @@
 """
 
 import os
-import datetime
```

Comment:
I think something funny has gone on with your PR here - there are many unrelated changes.

Could you try rebasing (or closing and opening a new PR if you prefer) to get just your Azure related changes?

Reply:
Looking at the conflicts, something went wrong. Maybe cherry-pick your commit on top of a clean master?

Reply:
Thank you. I will close this PR and create a new one with the correct files.